Makefile

```diff
@@ -0,0 +1,27 @@
+PG_CONFIG ?= $(shell which pg_config)
+DISTNAME = vchord
+# Extract DISTVERSION from vchord.control
+CONTROL_FILE = vchord.control
+DISTVERSION := $(shell sed -n "s/^[[:space:]]*default_version[[:space:]]*=[[:space:]]*'\([^']*\)'/\1/p" $(CONTROL_FILE))
+
+all: package
+
+# Install vchord into the PostgreSQL cluster identified by pg_config.
+.PHONY: install
+install:
+    @PGRX_PG_CONFIG_PATH="$(PG_CONFIG)" cargo make install -i ./build/raw
```

Comment:
It's 4-spaces, not a tab.

And don't use `@`.

Reply:
fixed